src/client/pps/util.go

```diff
@@ -88,7 +80,7 @@ func InputBranches(input *Input) []*pfs.Branch {
 		if input.Git != nil {
 			result = append(result, &pfs.Branch{
 				Repo: &pfs.Repo{Name: input.Git.Name},
-				Name: input.Atom.Branch,
+				Name: input.Git.Branch,
```

Comment:
@jdoliner I think this is a bug I happened across while working on this - could you confirm that the old code was wrong?

Reply:
That is most certainly a bug, and would be likely be a source of segfaults.